torch/_dynamo/utils.py

```diff
@@ -1057,6 +1057,9 @@ def run_node(output_graph, node, args, kwargs, nnmodule):
             return nnmodule(*args, **kwargs)
         elif op == "get_attr":
             return output_graph.get_submodule(node.target)
+        elif op == "placeholder":
+            if "example_value" in node.meta:
+                return node.meta["example_value"]
```

Comment:
This now returns none if there no example_value in node.meta, can we keep it going to RuntimeError in these cases? or, if you prefer, assert that we have an example_value for placeholder.

Reply:
I think it is fine because it would raise an Exception as we don't return in the try/except block.

Reply:
https://github.com/pytorch/pytorch/blob/6bf0e3b697ce688bc8325440dea3b51fea571c3d/torch/_dynamo/utils.py#L1060